### PR TITLE
chore(schema): add tracing for schema extraction

### PR DESCRIPTION
**Pull request by sgulseth**
### Description

adds tracing for extracting schema

### What to review

naming + attributes

### Testing

N/A

### Notes for release

N/A - no notes needed

### CODE REVIEW

**Comment by sgulseth** [comment]
* **#6019** <a href="https://app.graphite.dev/github/pr/sanity-io/sanity/6019?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `next`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @sgulseth and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/sanity/src/_internal/cli/actions/schema/extractSchema.telemetry.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import {defineTrace} from '@sanity/telemetry'
+
+interface SchemaExtractedTraceAttrubutes {
+  allTypes: number
+  documentTypes: number
+  types: number
```
Comment (joneidejohnsen) [inline packages/sanity/src/_internal/cli/actions/schema/extractSchema.telemetry.ts:6]: Same with this one – `schemaTypes`?

File: packages/sanity/src/_internal/cli/actions/schema/extractSchema.telemetry.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import {defineTrace} from '@sanity/telemetry'
+
+interface SchemaExtractedTraceAttrubutes {
+  allTypes: number
```
Comment (joneidejohnsen) [inline packages/sanity/src/_internal/cli/actions/schema/extractSchema.telemetry.ts:4]: Is this the number of types? If so, I'd call it `numberOfTypes` or `typesCount`

**Review by joneidejohnsen — APPROVED** [review]
LGTM

**Review by binoy14 — APPROVED** [review]
makes sense to me

**Pull request merged**